mmv1/third_party/terraform/resources/resource_cloudfunctions_function.go

```diff
@@ -146,6 +151,14 @@ func resourceCloudFunctionsFunction() *schema.Resource {
 				},
 			},
 
+			"docker_registry": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "CONTAINER_REGISTRY",
```

Comment:
Generally we try to avoid having a hard-coded default on the TF side, because we can never change it without breaking user configurations. (Because that would automatically change all existing user configs that were using the old default.) Instead, use an Optional + [Computed](https://www.terraform.io/plugin/sdkv2/schemas/schema-behaviors#computed) field (if there is an API-side default enforced) or make the field required.

Reply:
Adjusted this to Optional + Computed, as there is a defaulting logic on the API.